### DISCUSSION TITLE
php.ini change in the existing instance

### DISCUSSION THREAD

**Discussion by ptolemayos2nd** [evertramos/docker-wordpress#64] [Q&A]
Hi everyone. I'm new to the docker, so I'm sorry for the lame question. 
I wanted to update the PHP parameters in the existing docker instance, so I changed the /conf.d/php.ini in the docker project directory, but I don't know how to make the change effective. 
It seems if I run again 

`docker-compose up -d`

doesn't work. Anyone more experienced can help?  Many thanks

**Comment by evertramos** [comment]
Hello @ptolemayos2nd!

Try to restart the container:

```bash
docker compose restart {YOUR-SERVICE-NAME} (new-site-wordpress if default)
```

If that does not work, run:

```bash
docker compose down
docker compose up
```

Hope it helps!
